doc/Development_Documentation/02_MVC/02_Template/02_Template_Extensions/README.md

```diff
@@ -28,7 +28,6 @@ All Twig extension functions are described below in detail, the following tables
 |------------------------------------------|-------------------------------------------------------------------|
 | `pimcore_cache()`         | Simple in-template caching functionality                                         |
 | `pimcore_device()`        | Helps implementing adaptive designs                                              |
-| `pimcoreglossary()`       | Deprecated: Extension to control the glossary engine                             |
 | `pimcore_glossary`        | Twig Filter: Apply filter on content to pass it to Glossary engine               |
```

Comment:
shall we move into a new table for all available filters, as this itable are all about functions as `abc()`